librz/bin/p/bin_elf.inc

```diff
@@ -1733,6 +1726,18 @@ static RzBinInfo *info(RzBinFile *bf) {
 	}
 	lookup_sections(bf, ret);
 	lookup_symbols(bf, ret);
+
+	RzBinGoInfo *go = rz_bin_file_find_golang_info(bf);
+	if (go) {
+		free(ret->compiler);
+		if (go->settings) {
+			ret->compiler = rz_str_newf("%s (%s)", go->version, go->settings);
+		} else {
+			ret->compiler = RZ_STR_DUP(go->version);
+		}
+		ret->lang = "go";
+		rz_bin_file_golang_info_free(go);
+	}
```

Comment:
this code is the same in all 3 plugins. Is it really plugin specific? Can we move it in rzbin itself and remove the golang function from the API?

Reply:
yes, it is very specific to golang only. i know it kinda sound like it is not, but you can compile go binaries on windows, linux and mac and the 3 file formats has different locations for this type of info.

Reply:
if you prefer, i can change `rz_bin_file_find_golang_info` to `rz_bin_file_golang_compiler()` and return only a `char*`